doc/arm_jogging/arm_jogging_tutorial.rst

```diff
@@ -0,0 +1,71 @@
+Real-Time Arm Jogging
+=====================
+
+This tutorial shows how to send real-time jogging commands to a ROS-enabled robot. Some nice features of the jogger are singularity handling and collision checking that prevents operator from breaking the robot.
+
+.. raw:: html
+
+        <object width="480" height="385"><embed
+        src="http://www.youtube.com/v/8sOucNloJeI&hl=en_US&fs=1&rel=0"
+        type="application/x-shockwave-flash" allowscriptaccess="always"
+        allowfullscreen="true" width="480"
+        height="385"></embed></object>
+
+Robot Requirements
+------------------
+The jogger streams an array of position or velocity commands to the robot controller. This is compatible with ros\_control ``position_controllers/JointGroupPositionControllers`` or ``velocity_controllers/JointGroupVelocityControllers``. You can check if these controllers are available for your robot by searching for the controller config file (typically named ``controllers.yaml``).
+
+Jogging may work on other robots that have a different control scheme but there is no guarantee. It has been tested heavily on UR robots using the [ur_modern_driver](https://github.com/ros-industrial/ur_modern_driver). The jogger currently does not limit joint jerk so may not be compatible with most heavy industrial robots.
+
+The jogger can publish ``trajectory_msgs/JointTrajectory`` or ``std_msgs/Float64MultiArray`` message types. This is configured in a yaml file (see ``config/sia5_simulated_config.yaml`` for an example).
+
+Getting Started
+---------------
+If you haven't already done so, make sure you've completed the steps in `Getting Started <../getting_started/getting_started.html>`_.
+
+You can test the jogger with this `Gazebo simulator <https://github.com/UTNuclearRoboticsPublic/motoman_project>`_. Gazebo is necessary because it provides ros\_control controllers. Clone the repo into your catkin workspace. Install dependencies:
```

Comment:
Is this really necessary to get the ros_control controllers? I don't like the idea of depending on UTNuclearRoboticsPublic. Would it be possible to use the Panda panda_moveit_config to keep dependencies within the ros-planning org?

Reply:
I can try to get the fake RViz trajectory controller working. That would be a lot simpler but I've never done it before.

On the other hand, if we use the Gazebo Motoman model, Professor Nishida said he would publicize the tutorial.  :+1:

Reply:
I tried for a couple hours today but I don't think the fake RViz controller is compatible with the jogger. It expects the entire trajectory to be sent in one chunk on `/execute_trajectory/goal`

Reply:
Thanks for looking into this. Have you looked into the [moveit_sim_controller](https://github.com/PickNikRobotics/moveit_sim_controller)? I don't want you to spend too much of your time on this. That said, maybe look into the moveit_sim_controller for 15 minutes to see if it has the necessary interfaces? If not, then I am happy to approve with the UTNuclearRoboticsPublic repo.

Reply:
The project you are referring to is not the Gazebo simulator, but it's a Motoman robot simulated in Gazebo.
Don't we have a Gazebo config for the Panda too? @mlautman Could you please try to get this tutorial running on the Panda? Only if somebody has successfully followed the tutorial steps, the tutorial is actually approved :wink:
I don't like the idea of adding yet another robot as well.

Reply:
@rhaschke If it will make you happy, I can send a video of running the tutorial steps on a fresh Melodic ROS install  :P